tests/fail-typecheck/custom-cost-wrong-type.egg

```diff
@@ -0,0 +1,3 @@
+;;; test that setting the cost of a function to a non i64 will be a type error
+(relation a ())
+(cost (a) "hi")
```

Comment:
did you mean unstable-cost here?
Our `fail-typecheck` tests are extremely brittle because they don't look for a specific error

Reply:
Yes thank you!